test/unit/authcache.spec.js

```diff
@@ -1,20 +1,18 @@
 // @flow
 
-import type { Logger } from '@verdaccio/types';
 import type { UserDataGroups } from '../../src/authcache.js';
 
 import { AuthCache, UserData } from '../../src/authcache.js';
+import logger from './partials/logger.js';
 
 
-const logger: Logger = {
-  error: jest.fn(),
-  info: jest.fn(),
-  debug: jest.fn(),
-  child: jest.fn(),
-  warn: jest.fn(),
-  http: jest.fn(),
-  trace: jest.fn()
+const TEST_USER: string = 'myUser';
```

Comment:
this is used within both files, so maybe a common file?

Reply:
I'm going to refactor this into the `partials` objects as verdaccio does.